src/main/java/sg/ncl/MainController.java

```diff
@@ -1930,6 +1930,9 @@ public String removeExperiment(@PathVariable String teamName, @PathVariable Stri
                         log.warn("remove experiment failed for Team: {}, Exp: {}", teamId, expId);
                         redirectAttributes.addFlashAttribute("message", error.getMessage());
                         break;
+                    case OBJECT_OPTIMISTIC_LOCKING_FAILURE_EXCEPTION:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) End this switch case with an unconditional break, return or throw statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS128)